src/main/java/com/minecolonies/coremod/client/gui/WindowHutFarmer.java

```diff
@@ -268,20 +268,20 @@ private static String calcDirection(@NotNull final BlockPos building, @NotNull f
 
         if (field.getZ() > building.getZ() + 1)
         {
-            dist = LanguageHandler.format("com.minecolonies.gui.workerHuts.farmerHut.South");
+            dist = LanguageHandler.format("com.minecolonies.coremod.gui.workerHuts.farmerHut.South");
         }
         else if (field.getZ() < building.getZ() - 1)
         {
-            dist = LanguageHandler.format("com.minecolonies.gui.workerHuts.farmerHut.North");
+            dist = LanguageHandler.format("com.minecolonies.coremod.gui.workerHuts.farmerHut.North");
         }
 
         if (field.getX() > building.getX() + 1)
         {
-            dist += LanguageHandler.format("com.minecolonies.gui.workerHuts.farmerHut.East");
+            dist += LanguageHandler.format("com.minecolonies.coremod.gui.workerHuts.farmerHut.East");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Prefer StringBuffer over += for concatenating strings [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AUseStringBufferForStringAppends)